### ISSUE TITLE
java-mp stack missing mpHealth enable, failing health probe

### ISSUE THREAD

**Issue by dacleyra** [kabanero-io/kabanero-collection#62]
The java microprofile default stack is missing mpHealth enablement, thereby causing the pod to fail start when deployed as an `AppsodyApplication` with `createKnativeService: true`

**Comment by kilnerm** [comment]
@dacleyra I have been able to start and deploy the microprofile collection without the knative setting and I can see the health endpoint fine. Can you share the logs from your knative deployment that show mpHealth not starting?

I am presently unable to deploy using knative on the latest appsody code to try to recreate this issue. (But am working on getting this resolved.)

**Comment by seabaylea** [comment]
If this is only failing for Knative serving, it’s more likely the issue is in the appsody-operator. @arthurdm can you take a look?

**Comment by arturdzm** [comment]
I think it might have been older version of Appsody CLI or the stack, I tried 0.4.1 and mpHealth is there

**Comment by kilnerm** [comment]
@dacleyra, could you verify you see this resolved with the latest collections / appsody cli and if so close the issue please?

Thanks.

**Comment by kvijai82** [comment]
We spent some time chasing this with a bunch of folks last night and it turned out that image the the pipeline was building using Kaniko had some bugs which caused it point to the wrong server.xml which did not have the right features.  We hacked a workaround to test it and got it to work in knative serving.  

Opened this issue to see if we can make a change in the Dockerfile in the MP stack temporarily til we get off Kaniko and move to Buildah.

https://github.com/appsody/stacks/issues/303
